package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "toezicht-search-query-management-service",
-  "version": "0.3.6",
+  "version": "0.3.7-rc.1",
```

Comment:
Could you reverse this commit? That way we can squash merge the PR and it won't show in the history anymore.